Collections/Map Utilities/map.alias

```diff
@@ -312,53 +342,44 @@ if args.last('mapload') or args.last('loadmap'):
      desc.append(f"- Size: `{mapsize}`")
     if mapoptions:
      desc.append(f"- Options: `{mapoptions}`")
-     if mapoptions[0].isdigit():
+     if 'z' in mapoptions or mapoptions[0].isdigit():
       zoom = ""
-      for char in mapoptions:
-       if char.isdigit() or char == ".":
-        zoom += char
-       else:
-        break
-      desc.append(f" -- Zoom set to {zoom}x")
+      if 'z' in mapoptions:
+       zoom = ",".join([char for char in mapoptions.split('z')[1][0:3] if char.isdigit() or char == "."]).replace(",", " ")
+      else:
+       zoom = ",".join([char for char in mapoptions[0:3] if char.isdigit() or char == "."]).replace(",", "")
+      desc.append(f' - Zoom set to {zoom}x')
      if 'd' in mapoptions:
-      desc.append(f" -- Dark Mode")
+      desc.append(f" - Dark Mode")
```

Comment:
Why did all these -'s get removed? They were to indicate that these settings were a subset of the map options

Reply:
I honestly didn't realized they had two in the beginning. When trying to refactor that section of the code, I was going off of  normal one dash look since it shows as a header "Options (inputted options)" above all the individually described options.